script/installation/packages.sh

```diff
@@ -55,26 +66,43 @@ if [ "$DISTRO" = "UBUNTU" ]; then
         FORCE_Y=""
     fi
 
+    FORCE_Y=""
+    PKG_CMAKE="cmake"
+    PKG_LLVM="llvm-3.7"
+    PKG_CLANG="clang-3.7"
+
+    # Fix for cmake name change on Ubuntu 14.x and 16.x plus --force-yes deprecation
+    if [ "$MAJOR_VER" == "14" ]; then
+        PKG_CMAKE="cmake3"
+        FORCE_Y="--force-yes"
+    fi
+    # Fix for llvm on Ubuntu 17.x
+    if [ "$MAJOR_VER" == "17" ]; then
+        PKG_LLVM="llvm-3.9"
+        PKG_CLANG="clang-3.8"
```

Comment:
Can you explain why the package is *llvm-3.9*, but we use *clang-3.8*? Or is this a typo?

I also want to remind that I still have this weird behavior on *Ubuntu 16.04* when using *llvm-3.8*, that the build succeeds, but LLVM creates unsupported processor instructions during runtime. We maybe should check if this is only a problem for me.

Reply:
llvm-3.9 was the only version that is available on their repository. I looked for a PPA but I couldn't find it.

This is running correctly on my new workstation at home. I am going to merge this now and then we can revisit when Ubuntu 18.04 LTS comes out.